packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/Friend.Serialization.cs

```diff
@@ -5,32 +5,31 @@
 using System;
 using System.ClientModel.Primitives;
 using System.Text.Json;
+using UnbrandedTypeSpec.Models;
 
-namespace UnbrandedTypeSpec.Models
+namespace UnbrandedTypeSpec
```

Comment:
Why is this remove `.Models` since use model namespace has a default of true?

Reply:
This is an issue on this PR indeed, but this has been resolved in the other refactor PR.